Source/Core/DolphinQt/QtUtils/NonAutodismissibleMenu.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QMenu>
+
+namespace QtUtils
+{
+
+/// A menu widget based on \c QMenu that will not be automatically dismissed when one of its
+/// \em checkable actions are triggered.
```

Comment:
This comment has funky stuff going on?

Reply:
I guess you mean the Doxygen commands?

I saw there are some preexisting cases in Dolphin, so I thought it'd make sense to add them in the class documentation. What are the general feelings on that?

Reply:
I would prefer to leave it out unless you plan to set up doxygen.